pagopa/pn-frontend
# PN-13438 - Rework special contacts section

## Short description
This rework is focused on graphical improvements ([Figma](https://www.figma.com/design/DlzvNQzkecy7L2ym40OTge/Domicilio-digitale---08%2F11?node-id=2711-32652&t=gynGqfNUOYJN1hbX-4)). The functionality remains the same, but with one key change: it is no longer possible to add a new special address directly from this section. Instead, you will need to use a wizard that will be implemented in the future.

From this section, you can only edit or delete existing special addresses:
- If the address is a PEC, you can edit or delete it. [Figma](https://www.figma.com/design/DlzvNQzkecy7L2ym40OTge/Domicilio-digitale---08%2F11?node-id=2601-30234&t=gynGqfNUOYJN1hbX-4)
- If the contact is a SERCQ_SEND, you can only deactivate it.

## List of changes proposed in this pull request
- Graphical fixes
- Remove some useless logics

## How to test
Start PF/PG app and ensure that you have at least one special address. Try editing or deleting it, and verify that everything works well.
